src/modules/previewpane/powerpreview/dllmain.cpp

```diff
@@ -0,0 +1,233 @@
+#include "pch.h"
+#include <interface/powertoy_module_interface.h>
+#include <interface/lowlevel_keyboard_event_data.h>
+#include <interface/win_hook_event_data.h>
+#include "trace.h"
+#include <../common/settings_objects.h>
+
+extern "C" IMAGE_DOS_HEADER __ImageBase;
+
+BOOL APIENTRY DllMain(HMODULE hModule, DWORD ul_reason_for_call, LPVOID lpReserved)
+{
+    switch (ul_reason_for_call)
+    {
+    case DLL_PROCESS_ATTACH:
+        Trace::RegisterProvider();
+        break;
+    case DLL_THREAD_ATTACH:
+    case DLL_THREAD_DETACH:
+        break;
+    case DLL_PROCESS_DETACH:
+        Trace::UnregisterProvider();
+        break;
+    }
+    return TRUE;
+}
+
+// PowerToy Winodws Explore File Preview Settings.
+struct PowerPreviewSettings
+{
+    bool svgPreview_in_explr_IsEnabled = true;
+    bool svgPreview_in_prevpane_IsEnabled = true;
+    bool mdPreview_in_prevpane_IsEnabled = true;
+
+    const std::wstring EXPLR_RENDRSVG_BOOL_TOGGLE = L"explr_rendrsvg_bool_toggle";
+    const std::wstring PREVPANE_RENDRSVG_BOOL_TOGGLE = L"prevpane_rendrsvg_bool_toggle";
+    const std::wstring PREVPANE_RENDMD_BOOL_TOGGLE = L"prevpane_rendrmd_bool_toggle";
+
+} PowerPreviewSettings;
+
+// Implement the PowerToy Module Interface and all the required methods.
+class FileExplorer : public PowertoyModuleIface
```

Comment:
`FileExplorer` sounds too generic and we can name it as `PowerPreview` similar to other powertoys module.